aicspylibczi/CziFile.py

```diff
@@ -109,6 +109,17 @@ def dims_shape(self):
         """
         return self.reader.read_dims()
 
+    @property
+    def pixel_type(self):
+        """
+        The pixelType of the images. If the pixelType is different in the different subblocks it returns Invalid.
+
+        Returns
+        -------
+        A string containing the name of the type of each pixel. If inconsistent it returns invalid.
+        """
+        return self.reader.pixel_type()
```

Comment:
Can czi have different pixel types for different scenes?

Reply:
it's set at the subblock level so it appears so.